### PR TITLE
Update jcanvas-handles.js

### PR DESCRIPTION
**Pull request by imarian97**
For rectangles I fixed the bug that occurs when you set "fromCenter:false" and also I implemented the minWidth and minHeight functionality.

### CODE REVIEW

**Comment by caleb531** [comment]
First of all, thanks for making a pull request for this issue.

I'm having trouble finding the logical differences in the code, because the indentation for the entire file as been modified. Could you please revert your whitespace/indentation changes so I can more easily review the logic that actually changed?

Thanks,
Caleb

**Comment by imarian97** [comment]
Sorry about that, I fixed the problem.

File: plugins/jcanvas-handles.js [hunk 1]
```diff
@@ -109,10 +109,31 @@ function addRectHandle( $canvas, parent, px, py ) {
 		drag: function ( layer ) {
 			var parent = layer._parent;
 			
+			if (parent.width + layer.dx * layer._px < parent.minWidth) {
```
Comment (caleb531) [inline plugins/jcanvas-handles.js]: I realize that JavaScript's operator precedence will make this irrelevant, but for clarity, I would prefer that you wrap your multiplication in parens like so:

```javascript
if (parent.width + (layer.dx * layer._px) < parent.minWidth) {
```

**Comment by caleb531** [comment]
Ah, much better—thanks for that. I've tested your code with the existing examples on the [plugin page for jCanvas Handles](http://calebevans.me/projects/jcanvas/plugins/handles/), and they all seem to work. Nice job.

However, I have one final request before I'm ready to merge (forgive me for failing to mention this earlier). I have chosen to abide by the [jQuery JavaScript Style Guide](http://contribute.jquery.org/style-guide/js/) for jCanvas *and* its official plugins (which includes Handles). Therefore, if you could please tweak your changes to abide by the style guidelines, I can merge this PR directly into `master`.

In general, you only need to ensure that:

* Your groups of parens are padded with a single space on each side (*e.g.* `if ( a === 3 )` instead of `if (a === 3)`)
* Key-value pairs have a space to the right of the colon, but not the left (*e.g.* `key: "value"` instead of `key : "value"`)

Also see my comment on one particular line in your code.

Other than that, your pull request is golden. :)

Thanks,
Caleb

**Comment by imarian97** [comment]
I'm writing you this message because I think you didn't see my last commit.

**Comment by caleb531** [comment]
Ah, yes. Apologies for missing that.

Thanks for the reminder. I'll merge that in later today.

Caleb

**Comment by caleb531** [comment]
Changes look entirely solid. Time to merge!

**Pull request merged**